src/shared/test/java/com/couchbase/lite/DatabaseTest.java

```diff
@@ -223,7 +223,14 @@ public void testCreateWithDefaultConfiguration() throws CouchbaseLiteException {
 
     @Test
     public void testCreateWithSpecialCharacterDBNames() throws CouchbaseLiteException {
-        Database db = openDatabase("`~@#$%^&*()_+{}|\\][=-/.,<>?\":;'ABCDEabcde");
+        Database db;
+        if (System.getProperty("os.name").toLowerCase().contains("windows")) {
```

Comment:
We may need to extract this into a util function if there are more than one places that need to do this.

Reply:
I can see a test with `testSaveNewDocWithSpecialCharactersDocID`, which seems to be working fine. Since it is document-id. But in this case, this is database name(folder name).  Other than that, I couldn't find any place.